backend/src/crd/controller/viewer/reconciler/reconciler.go

```diff
@@ -180,6 +181,12 @@ func setPodSpecForTensorboard(view *viewerV1beta1.Viewer, s *corev1.PodSpec) {
 		// when https://github.com/kubeflow/pipelines/issues/2514 is done
 		// "--bind_all",
 	}
+
+	matched, _ := regexp.MatchString(`tensorflow/tensorflow:2.`, view.Spec.TensorboardSpec.TensorflowImage)
```

Comment:
nit: you are not using any regex features, just using string.startsWith is simpler

Reply:
Thanks for the review and suggestion! I will re-format the code today.